usgs/magpy
# Add option to calculate baselines based on USGS absolutes data

This implements the new `readJSONABS` function that will parse USGS json files. Clicking the *Load USGS data* button will create a DI data list based on the geomag baselines web service. Extending the start and end of the time range (based on the plotted data) will ensure that the available unique baseline data covers the time range of the plotted data.
To test:
- Open a USGS URL or file
- Navigate to the DI page
- Click Load USGS data
   - choose whether or not to extend the time range (not required for all data sets, but recommended)
- Click Analyze
- Stream Operations -> Select Active Stream...
    - Choose the original data
- Navigate to the Analysis page
- Click Baseline
   - To change type of fit: Options -> Basic initialization parameter
   - polynomial appears to work best
- Navigate to the Stream page
- Click Baseline Corr
